frontend/src/pages/posts/comments/CommentCard.jsx

```diff
@@ -0,0 +1,18 @@
+// NPM Packages
+import React from "react";
+
+export default function Comment({ postId, comment, onDeleteClick }) {
```

Comment:
Should a component name also be CommentCard?

Reply:
yes, good spot, thanks

Reply:
done